### PR TITLE
Add show methods for GitRebase and RebaseOperation

### PR DESCRIPTION
**Pull request by kshyatt**
Now you can see what `RebaseOperation` is happening at any one step!

### CODE REVIEW

File: base/libgit2/rebase.jl [hunk 1]
```diff
@@ -41,6 +41,11 @@ function Base.next(rb::GitRebase)
     return unsafe_load(rb_op_ptr_ptr[])
 end
 
+function Base.show(io::IO, rb::GitRebase)
+    println(io, "GitRebase:")
+    println(io, "Number of operations to apply: $(count(rb))")
```
Comment (ararslan) [inline base/libgit2/rebase.jl]: Could you just do `"Number ...: ", count(rb)` to avoid the interpolation? Might be very marginally faster

**Comment by ararslan** [comment]
This is really cool! Nice work!

**Comment by tkelman** [comment]
Is PkgDev or any other code expecting that getindex to be returning nothing, or the flags to be constants instead of an Enum?

**Comment by kshyatt** [comment]
PkgDev doesn't include rebasing at all, as far as I can tell.

**Comment by kshyatt** [comment]
A search for the name of the constants/enum shows nothing in Julia on GH, I can't see into people's private repos, of course.

File: base/libgit2/rebase.jl [hunk 2]
```diff
@@ -41,6 +43,11 @@ function Base.next(rb::GitRebase)
     return unsafe_load(rb_op_ptr_ptr[])
 end
 
+function Base.show(io::IO, rb::GitRebase)
+    println(io, "GitRebase:")
+    println(io, "Number: ", count(rb))
+    println(io, "Currently performing operation: $(current(rb)+1)")
```
Comment (ararslan) [inline base/libgit2/rebase.jl]: Should remove the interpolation here too for consistency since you did so above

**Comment by kshyatt** [comment]
Good to go if CI passes?

**Review by ararslan — APPROVED** [review]

**Comment by kshyatt** [comment]
It's the `spawn` failure again.

**Pull request merged**